Frameworks/UIKit/WYPopoverController.mm

```diff
@@ -2634,8 +2639,12 @@ - (void)dismissPopoverAnimated:(BOOL)aAnimated
         }
 #endif
     };
+
+    UIPopoverPresentationController* popoverPresentationController = [viewController popoverPresentationController];
     
     void (^completionBlock)() = ^() {
+        // Keep popoverPresentationController alive via block-capture during teardown.
+        (void)popoverPresentationController;
```

Comment:
I was suggesting something like: UIPopoverPresentationController* blockCapture = popoverPresentationController;

---
In reply to: [109064009](https://github.com/Microsoft/WinObjC/pull/2195#discussion_r109064009) [](ancestors = 109064009)